kyo-data/shared/src/main/scala/kyo/Show.scala

```diff
@@ -0,0 +1,93 @@
+package kyo
+
+import kyo.Schedule.done
+import scala.language.implicitConversions
+
+/** Provides string representation of a type as an alternative to .toString. Needed for customizing how to display opaque types
+  */
+trait Show[A]:
```

Comment:
```suggestion
abstract class Show[A]:
```

Reply:
No objections to the change, but out of curiosity what's the reason for the change?

Reply:
Traits compile to interfaces, which introduce interface dispatch. It won't matter much here, but it's just to promote consistency throughout the codebase.